File: java/HelloWorld.java
public class HelloWorld {
    public static void main(String[] args) {
        System.out.println("Hello World!");

        int someNumber = 3;
        int answer = add(2, someNumber);
        System.out.println("2 + 3 = " + answer);
    }

    public static int add(int number1, int number2) {
        return number1 + number2;
    }
}

File: go/hello_world.go
package main

import "fmt"

func add(x int, y int) int {
    return x + y
}

func main() {
    fmt.Println("Hello, World!")

    var someNum = 3
    var answer = add(2, someNum)
    fmt.Println("2 +", someNum, "=", answer)
}
